client/transport/src/test/java/org/elasticsearch/transport/client/PreBuiltTransportClientTests.java

```diff
@@ -54,7 +56,7 @@ public void testInstallPluginTwice() {
                 new PreBuiltTransportClient(Settings.EMPTY, plugin);
                 fail("exception expected");
             } catch (IllegalArgumentException ex) {
-                assertEquals("plugin is already installed", ex.getMessage());
+                assertTrue(ex.getMessage().startsWith("plugin already exists: "));
```

Comment:
put the message as the first arg on assertTrue so we know what it was when it fails.